libs/application/template-api-modules/src/lib/modules/templates/health-insurance/health-insurance.utils.ts

```diff
@@ -119,6 +119,11 @@ export const transformApplicationToHealthInsuranceDTO = (
     previousIssuingInstitution:
       extractAnswer(application.answers, 'formerInsurance.institution') ?? '',
     isHealthInsuredInPreviousCountry:
+      extractAnswer(application.answers, 'formerInsurance.registration') ==
```

Comment:
is there a reason why the check is `==` and not `===`? :)

Reply:
I also don't like my own naming in the frontend of this field (`formerInsurance.registration`). I will change it in a separate PR and make sure I update in this file as well.